core/field_angle.js

```diff
@@ -280,15 +281,16 @@ Blockly.FieldAngle.prototype.dropdownCreate_ = function() {
 
   // The angle picker is different from other fields in that it updates on
   // mousemove even if it's not in the middle of a drag.  In future we may
-  // change this behaviour.  For now, using bindEvent_ instead of
-  // bindEventWithChecks_ allows it to work without a mousedown/touchstart.
+  // change this behaviour.
   this.clickWrapper_ =
-      Blockly.bindEvent_(svg, 'click', this, this.hide_);
-  this.moveWrapper1_ =
-      Blockly.bindEvent_(circle, 'mousemove', this, this.onMouseMove);
-  this.moveWrapper2_ =
-      Blockly.bindEvent_(this.gauge_, 'mousemove', this, this.onMouseMove);
-
+      Blockly.bindEventWithChecks_(svg, 'click', this, this.hide_);
```

Comment:
Does the change to bindEventWithChecks work for the case of changing it on mousemove without a drag?

Reply:
Yes, when using mouseevents / pointerevents.

Reply:
But I assume you're talking about the 'mousemove' two lines below? This one just hides on click.

Reply:
That's right.  The comment above references it as well.